.github/ISSUE_TEMPLATE/new_bug_report.yaml

```diff
@@ -0,0 +1,151 @@
+- type: markdown
+  attributes:
+    value: |
+      ## Environment
+- type: input
+  attributes:
+    label: How did you install the Amplify CLI?
+    description: "For example: npm, yarn, etc."
+- type: input
+  attributes:
+    label: If applicable, what version of Node.js are you using?
+    description: You can use `node -v` to check the node version on your system.
+- type: input
+  attributes:
+    label: Amplify CLI Version
+    description: You can use `amplify -v` to check the amplify cli version on your system.
+    placeholder: "4.44.2"
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: What operating system are you using?
+    description: "For example: Mac, Windows, Ubuntu."
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: Amplify Categories
+    description: What [Amplify categories](https://github.com/aws-amplify/amplify-cli#category-specific-commands) are being used?
+    multiple: true
+    options:
+      - auth
+      - storage
+      - function
+      - api
+      - analytics
+      - hosting
+      - notifications
+      - interactions
+      - predictions
+      - xr
+      - None
+  validations:
+    required: true
+- type: markdown
+  attributes:
+    value: |
+      ## Details
+- type: dropdown
+  attributes:
+    label: Amplify Commands
+    description: If applicable, what Amplify CLI commands (`amplify ...`) are surfacing the issue?
+    multiple: true
+    options:
+      - configure 
+      - init 
+      - configure project
+      - add 
+      - update
+      - push 
+      - pull
+      - publish
+      - status 
+      - serve 
+      - delete
+      - codegen 
+      - env 
+      - None
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: A clear and concise description of what the bug is. Markdown formatting supported.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Expected behavior
+    description: A clear and concise description of what you expected to happen. Markdown formatting supported.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Reproduction steps
+    description: | 
+      How do you trigger this bug? Please walk us through it step by step. 
+      Screenshots can be provided in the issue body below. Markdown formatting supported.
+    value: |
+      1. 
+      2.
+      3.
+      ...
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Log output
+    description: |
+      For example, error messages, or stack traces.
+
+      Include any relevant log output under:
+      
+      ```
+      ~/.amplify/logs/amplify-cli-<issue-date>.log
+      ``` 
+      
+    value: |
+      ```
+      # Put your logs below this line
+
+      ```
+- type: markdown
+    attributes:
+      value: |
+        ## Additional information
+- type: markdown
+  attributes:
+    value: >
```

Comment:
I think we're missing a `textarea` input for this.

Reply:
The area directly below this in the template is the general open-ended GH wysiwyg.